packages/spor-elm/src/Spor/LineTag/LineIcon.elm

```diff
@@ -152,115 +152,140 @@ iconColor variant =
 
 icon : Variant -> Size -> Svg msg
 icon variant size =
-    Svg.fromUnstyled <|
+    let
+        trainIcon size_ =
+            Icon.icon size_ Icon.Fill Icon.Train
+
+        expressBusIcon size_ =
+            Icon.icon size_ Icon.Fill Icon.ExpressBus
+
+        busIcon size_ =
+            Icon.icon size_ Icon.Fill Icon.Bus
+
+        ferryIcon size_ =
+            Icon.icon size_ Icon.Fill Icon.Ferry
+
+        subwayIcon size_ =
+            Icon.icon size_ Icon.Fill Icon.Subway
+
+        tramIcon size_ =
+            Icon.icon size_ Icon.Fill Icon.Tram
+
+        altTransportIcon size_ =
+            Icon.icon size_ Icon.Fill Icon.AltTransport
+
+        walkIcon size_ =
+            Icon.icon size_ Icon.Fill Icon.Walk
+    in
+    Icon.toHtml <|
         case ( variant, size ) of
             ( LocalTrain, Sm ) ->
-                Transportation.trainFill18X18 []
+                trainIcon Icon.Size18
 
             ( LocalTrain, Md ) ->
-                Transportation.trainFill24X24 []
+                trainIcon Icon.Size24
 
             ( LocalTrain, Lg ) ->
-                Transportation.trainFill30X30 []
+                trainIcon Icon.Size30
```

Comment:
Hva med å lage en sizeToIconSize i let...in, så kan man i stedet si 
```
( LocalTrain, size_ ) ->
   trainIcon <| sizeToIconSize size_
   
( RegionTrain, size_ ) ->
  trainIcon <| sizeToIconSize size_
```
osv..? 😄

Reply:
Hvis et case også har unikt ikon for sin variant kan egentlig ikonfunksjone bare ligge i den casen også!